[KOGITO-5300] Updating documentation of addons modules (#1463)
kiegroup/kogito-runtimes: addons/common/tracing/README.md

# Kogito Tracing Addon

> Adds the capability of tracing to Kogito projects. At this time, we only support DMN services tracing. To find out more

Reviewer: ```suggestion
The Kogito Tracing addon adds tracing capability to Kogito projects. Currently, we support DMN services tracing only. To find out more
```

> about this capability, please check

Reviewer: ```suggestion
about this capability, see 
```

our [official documentation](https://docs.jboss.org/kogito/release/latest/html_single/#con-trusty-service_kogito-configuring).

## Examples

> Please check the following examples to explore more about this capability:

Reviewer: ```suggestion
See the following examples to explore more about this capability:
```
